app/layouts/DashboardLayout/TopBar/NavBar.tsx

```diff
@@ -84,7 +84,7 @@ const NavBar: FC<NavBarProps> = () => {
             icon={<Icon height={32} width={32} color={color} />}
             label={t(label)}
             key={label}
-            wrapped // FK
```

Comment:
Lol Is this something you did to test me? I will absolutely miss minor things like this.

Reply:
Not that! I usually mark experimental code with `// FK` so it will be easier for me to search and remove... though on occasion I've been known to miss the last part!

Reply:
Hahah now I'm gonna start putting random comments in the code to see if you can spot them. 
![image](https://user-images.githubusercontent.com/4346395/110192028-c0ce3400-7df9-11eb-9453-7d732bc46df9.png)